index.html

```diff
@@ -62,7 +62,7 @@ <h3 class="SD">I'm a Software Developer</h3>
   </div>
 </section>
 
-    <section class="Recent-works">
+    <section class="recent-works">
```

Comment:
- [ ] Please Fix the icons. You need to add a `.` to the beginning of the icon file path as shown below `<a href="#"><img src="./images/medium.png" class="contact-icon"></a>` You should always start your relative path with a `.` You can read more about it [here](https://stackoverflow.com/questions/66887694/in-the-file-path-what-does-mean) 
![links-1](https://user-images.githubusercontent.com/99656687/213862339-be942d3f-4a82-4275-a105-49f18e3aedf5.png)